src/components/Table/TableRow.jsx

```diff
@@ -0,0 +1,17 @@
+import styled from '@xstyled/styled-components'
+import { th } from '@xstyled/system'
+
+const TableRow = styled.tr`
+  padding: 16px;
+  min-height: 48px;
+  max-height: 48px;
```

Comment:
Por que utilizar min-height e max-height ao invés de utilizar apenas height? Existe algum motivo específico?

Reply:
Eu uso pra forçar a altura.